api/src/main/java/io/opentelemetry/trace/attributes/SemanticAttributes.java

```diff
@@ -142,18 +142,66 @@
   public static final LongAttributeSetter MESSAGE_UNCOMPRESSED_SIZE =
       LongAttributeSetter.create("message.uncompressed_size");
   /** Database type. For any SQL database, "sql". For others, the lower-case database category. */
-  public static final StringAttributeSetter DB_TYPE = StringAttributeSetter.create("db.type");
-  /** Database instance name. */
-  public static final StringAttributeSetter DB_INSTANCE =
-      StringAttributeSetter.create("db.instance");
+  public static final StringAttributeSetter DB_SYSTEM = StringAttributeSetter.create("db.system");
+  /** Database name. */
+  public static final StringAttributeSetter DB_NAME = StringAttributeSetter.create("db.name");
+  /**
+   * The connection string used to connect to the database. It's recommended to remove embedded
+   * credentials.
+   */
+  public static final StringAttributeSetter DB_CONNECTION_STRING =
+      StringAttributeSetter.create("db.connection_string");
   /** Database statement for the given database type. */
   public static final StringAttributeSetter DB_STATEMENT =
       StringAttributeSetter.create("db.statement");
+  /** Database operation that is being executed. */
+  public static final StringAttributeSetter DB_OPERATION =
+      StringAttributeSetter.create("db.operation");
   /** Username for accessing database. */
   public static final StringAttributeSetter DB_USER = StringAttributeSetter.create("db.user");
   /** JDBC substring like "mysql://db.example.com:3306" */
   public static final StringAttributeSetter DB_URL = StringAttributeSetter.create("db.url");
```

Comment:
`db.url` was removed from the semantic conventions

Reply:
Good point, but this is required when making a connection.  Maybe it was missing from the database.md?

Reply:
I just realized i added db_connection_string, i can remove db_url.  Please confirm if that's what you meant.  @thisthat

Reply:
Deleted.